### ISSUE TITLE
Ubuntu 20.04 is not supported.

### ISSUE THREAD

**Issue by aplsms** [threatstack/threatstack-ansible#68]
during installation I have an error:

```
The full traceback is:
  File "/tmp/ansible_apt_repository_payload_ew1nuf_7/ansible_apt_repository_payload.zip/ansible/modules/packaging/os/apt_repository.py", line 548, in main
  File "/usr/lib/python3/dist-packages/apt/cache.py", line 591, in update
    raise FetchFailedException(e)
fatal: [10.79.4.111]: FAILED! => {
    "changed": false,
    "invocation": {
        "module_args": {
            "codename": null,
            "filename": null,
            "install_python_apt": true,
            "mode": null,
            "repo": "deb https://pkg.threatstack.com/v2/Ubuntu focal main",
            "state": "present",
            "update_cache": true,
            "validate_certs": true
        }
    },
    "msg": "apt cache update failed"
}
```
please create package for Ubuntu 20.04

**Comment by amberb617** [comment]
Hi @aplsms! Thank you so much for reaching out. While our 2.2.x agent does not yet support running on Ubuntu 20.x, it is our goal to support this for 2.3. I will be communicating a public release date for 2.3 soon, and will be sure to let you know.

**Comment by aplsms** [comment]
Any updates?

**Comment by amberb617** [comment]
Hey @aplsms we're targeting to have Linux Agent 2.2.3 available, with Ubuntu 20.04 support, in mid-July!

**Comment by aplsms** [comment]
Today is mid of July. From where i can download agent?
and it is fully: Ubuntu-20.04 deployment with security fixes blocked by company that care about security...

**Comment by amberb617** [comment]
Hi @aplsms we released the 2.2.3 Linux agent today with Ubuntu 20.04 support -- we will be updating our ansible scripts for this by the end of the week.

**Comment by aplsms** [comment]
Hello, I hope your las week already gone. is it available?

**Comment by amberb617** [comment]
Hi @aplsms, yes, you can use the Ansible script we have published for installation of 2.2.3  on Ubuntu 20.04. It will work! Please feel free to reach out to support@threatstack.com should you have any further questions. Thank you so much for your patience and feedback, and for using Threat Stack!
